controllers/userController.js

```diff
@@ -147,4 +148,45 @@ const deleteUserAddresses = async (req, res) => {
     res.status(500).send({ message: err })
   }
 }
-module.exports = { userRegister, login, getUserData, deleteUserData, limitUsersData, userAddress, deleteUserAddresses }
+
+const forgotPassword = async (req, res) => {
+  const { username } = req.body
+  try {
+    const userData = await userModel.findOne({ where: { username } })
+    if (userData) {
+      const passwordResetToken = jwt.sign({
+        data: `${userData.username} ${userData.password}`
```

Comment:
you didn't add the expiration time when creating the JWT token.  According to the assignment `Password reset token will only be active for 10 min.` @ashok-choudhary-0